.github/workflows/test_and_deploy.yml

```diff
@@ -40,7 +43,7 @@ jobs:
           conda install -y pyopencl pocl
           python -m pip install --upgrade pip
           python -m pip install setuptools wheel pytest pytest-cov pytest-benchmark pytest-qt PyQt5
-          pip install -e .
+          pip install --default-timeout=100 --use-deprecated=legacy-resolver -e ./core ./plugin
```

Comment:
why do we use a timeout?

Reply:
Its a connection issue, right?

Reply:
I'm not sure. I think the main issue was a dependency solving issue, but it doesn't seem to be happening anymore.